apps/src/gamelab/animationListModule.js

```diff
@@ -402,7 +403,12 @@ function blobToDataURI(blob, onComplete) {
 export function saveAnimations(onComplete) {
   return (dispatch, getState) => {
     const state = getState().animationList;
-    const changedAnimationKeys = state.orderedKeys.filter(key => !state.propsByKey[key].saved);
+    // Animations with a sourceUrl are referencing an external spritesheet and
+    // should not be saved - until an edit operation clears the sourceUrl.
+    // Also check the saved flag, so we only upload animations that have changed.
+    const changedAnimationKeys = state.orderedKeys.filter(key =>
+        !state.propsByKey[key].sourceUrl && state.propsByKey[key].blob &&
+        !state.propsByKey[key].saved);
```

Comment:
We used to throw an error when saving an animation that didn't have a blob (meaning it hasn't been asynchronously loaded into memory yet).  I've decided that's overkill, and it was causing issues in some scenarios where we want to change the metadata and then save immediately (as on clearing the level) - basically if an animation hasn't been loaded yet, it can't possibly have changed this session, so we don't need to save it.